src/sentry/static/sentry/app/views/releases/detail/organization/releaseAllEvents.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+import SentryTypes from 'app/sentryTypes';
+import GroupList from 'app/components/groupList';
+import {t} from 'app/locale';
+
+export default class ReleaseAllEvents extends React.Component {
+  static contextTypes = {
+    release: SentryTypes.Release,
+  };
+
+  render() {
+    let {orgId} = this.props.params;
+    return (
+      <div>
+        <div className="alert alert-block">
+          <Link
+            to={{
+              pathname: `/organizations/${orgId}/issues/`,
+              query: {query: 'release:' + this.context.release.version},
```

Comment:
Why is version not wrapped in `"` here? Should you URLencode `release.version`? It can contain URL unsafe data.

Reply:
Using `query` seems to automatically encode this for us

Reply:
I'm not exactly sure what the impact of wrapping in `"` is in the stream view (afaik both are valid) but since i'm not confident I've gone with the same implementation as the project releases version here https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/releases/detail/project/releaseAllEvents.jsx#L27